src/components/TagList/TagList.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="tags">
+    <i v-if="icon" class="pi" :class="icon" />
+    <Tag flat>
+      {{ internalTags }}
+      <template v-if="tagLengthLimit">
+        <Tooltip  placement="bottom">
+          <template v-slot:trigger>
+            <span class="tags__showTooltip">show</span>
+          </template>
+          <span class="tags__tooltipTag">{{ jointTags }}</span>
+        </Tooltip>
+      </template>
+    </Tag>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+import Tag from "@/components/Tag/Tag.vue";
+import Tooltip from "@/components/Tooltip/Tooltip.vue"
+
+export default defineComponent({
+  name: "TagList",
+  components: { Tag, Tooltip },
+  props: {
+    icon: {
+      type: String,
+      default: null
```

Comment:
Let's default this to `pi-label` since currently that's the only implementation of this we have planned.